integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -822,7 +821,7 @@ private void setupForAuthJourney() {
     private void setupForDocAppJourney() {
         registerClient(
                 CLIENT_ID, "test-client", List.of("openid", "doc-checking-app"), ClientType.APP);
-        handler = new AuthorisationHandler(configWithDocAppDecouple(true));
+        handler = new AuthorisationHandler(config());
```

Comment:
This can just be `configuration` now, right? Or we can get rid of the private static

Reply:
Yes, that'd make more sense